sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -106,17 +105,20 @@ case class DropFunction(
     isTemp: Boolean)(sql: String)
   extends NativeDDLCommand(sql) with Logging
 
-case class AlterTableRename(
+/** Rename in ALTER TABLE/VIEW: change the name of a table/view to a different name. */
+case class AlterTableAlterViewRename(
```

Comment:
`AlterTableAlterView*` is a bit contrived. Views are special tables in our backend, lets just call it `AlterTable*`?